ThePensionsRegulator.Frontend/Styles/_tpr-file-upload.scss

```diff
@@ -0,0 +1,36 @@
+﻿.govuk-file-upload {
+    font-size: 1rem;
+    outline: 3px solid transparent;
+    border-color: transparent;
+    box-shadow: inset 0 0 0 2px transparent;
+    padding: 3px 3px 1px 3px;
+    font-family: $govuk-font-family;
+    color: $govuk-text-colour
+}
+
+.govuk-file-upload::file-selector-button {
+    padding-top: 8px;
+    padding-bottom: 7px;
+    padding-left: $tpr-button-padding;
+    padding-right: $tpr-button-padding *2;
+    font-family: $govuk-font-family;
+    border: 1px solid $tpr-colour-royal-blue;
+    margin-right: 1rem;
+}
+
+.govuk-file-upload:focus-within::file-selector-button,
+.govuk-file-upload:focus::file-selector-button {
+    background-color: $tpr-colour-tangerine-yellow;
+    color: $tpr-colour-black;
+    border-left-color: transparent;
+    border-right-color: transparent;
+    border-top-color: transparent;
+}
+
+.govuk-file-upload:focus-within,
+.govuk-file-upload:focus {
+    outline: 3px solid #ffdd00;
+    border-color: #0b0c0c;
+    box-shadow: inset 0 0 0 2px #0b0c0c;
+    padding: 3px 3px 1px 3px;
```

Comment:
See earlier comment about removing this padding. You wouldn't need to re-specify it on focus anyway.

Reply:
Without the padding the Removed

Reply:
removed